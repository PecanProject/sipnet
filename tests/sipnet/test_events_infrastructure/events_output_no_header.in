2023  65  plant  10 5 4 3
2023  70  irrig  5 1
2023  200 harv   0.1 0.2 0.3 0.4
2024  65  plant  10 5 4 3
2024  70  irrig  5 0
2024  200 harv   0.2 0.3 0.4 0.1
