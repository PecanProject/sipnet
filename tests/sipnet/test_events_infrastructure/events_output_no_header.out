0    2024   65  plant    envi.plantLeafC=10.00,envi.plantWoodC=5.00,envi.fineRootC=4.00,envi.coarseRootC=3.00
0    2024   70  irrig    envi.soilWater=5.00
0    2024  200  harv     env.litter=12.40,envi.plantLeafC=-4.80,envi.plantWoodC=-3.20,envi.fineRootC=-4.80,envi.coarseRootC=-4.80
1    2024   65  plant    envi.plantLeafC=10.00,envi.plantWoodC=5.00,envi.fineRootC=4.00,envi.coarseRootC=3.00
1    2024   70  irrig    fluxes.immedEvap=2.50,envi.soilWater=2.50
1    2024  200  harv     env.litter=12.14,envi.plantLeafC=-10.32,envi.plantWoodC=-5.88,envi.fineRootC=-2.88,envi.coarseRootC=-2.48
